### PR TITLE
[Merged by Bors] - Add hexx

### PR DESCRIPTION
**Pull request by ManevilleF**
- Adds a *webp* instead of a a static image for [bevy_silk](https://github.com/ManevilleF/bevy_silk)
- Adds [hexx](https://github.com/ManevilleF/hexx) as a **Helper** asset ( maybe it should be put in 2D and/or 3D ?) It can generate hex grids and allows to generate meshes ([Associated thread](https://discord.com/channels/691052431525675048/918591326096850974/1069971738126975067))

### CODE REVIEW

**Comment by mockersf** [comment]
There is now a hard limit on file size at 2MB, and the smaller the better. Could you reduce the size of your gif, or try as a webp video?

**Comment by ManevilleF** [comment]
> There is now a hard limit on file size at 2MB, and the smaller the better. Could you reduce the size of your gif, or try as a webp video? ezgif.com seems to have an easy converter: https://im5.ezgif.com/tmp/ezgif-5-64a25eb2b3.webp

Done !

File: Assets/Helpers/hexx.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = "hexx"
+description = "Hexagon utils for 2D and 3D"
+link = "https://github.com/ManevilleF/hexx"
+image = "hexx.gif"
```
Comment (mockersf) [inline Assets/Helpers/hexx.toml:4]:
```suggestion
image = "hexx.png"
```

Comment (ManevilleF) [reply Assets/Helpers/hexx.toml]: Thank you !

**Review by mockersf — APPROVED** [review]

**Pull request closed without merging**